src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
```

Comment:
@amihaiemil after building the image, ssh to it, checkout sources and try out all commands used by rultor manually. I don't know any other method. If we don't do that, then the work is useless.